Rakefile

```diff
@@ -327,7 +326,7 @@ CONFIRMATION
     end
 
   system("powershell -nologo -noprofile -executionpolicy unrestricted Import-Module .\\scripts\\publishpkg.psm1;Publish-ChefPkg #{ENV["publishsettings"]} \"\'#{subscription_name}\'\" #{publish_uri} #{definitionXmlFile} #{postOrPut}")
-
+  File.open("C:\\publish.txt", "a+") {|f| f.write("\n#{args.extension_version}\t#{Time.now}\t #{File.dirname(__FILE__)}") }
```

Comment:
I don't really mind because this is an internal script, but it's a bad practice to hard code paths like `C:\\....` and using the root of the drive is particularly bad because it tends to have more restrictive permissions. I would either not put a path on it, so it will be dropped where you're running rake from, or put it in $temp.